docker-compose.yml

```diff
@@ -48,9 +48,6 @@ services:
       - db
       - redis-store
       - redis-cache
-      - statsd
-      - localstack-s3
-      - frontend
     command: web-dev
```

Comment:
Thanks!
See the followup commit I made. 

Now, when I `make run` it starts `localstack-s3` and `statsd` as daemons in the background (just like postgres and redis). 
But when I run `make test` it only starts the absolutely minimal. Now `make test` runs a lot faster than before. 
Best of both worlds.